### ISSUE TITLE
Make Sign out go to Multiaccount overview 'Your keys' screen

### ISSUE THREAD

**Issue by hesterbruikman** [status-im/status-react#9902]
# Problem
When the user has set `Save password`, going to the `unlock view` can result in an endless loop of sign out>unlock, making it impossible to switch accounts. This scenario is currently counteracted (does not occur), because auth-methods are erased on `Sign out`. However, erasing auth-methods on sign out causes other issues:
- Requiring Keycard to open a Keycard multiaccount, in spite of having set `Don't ask for Keycard`, see #9763
- The user having to re-enable biometric unlock (fingerprint or faceID) without being informed that they are required to do so when they sign out

## Implementation
To resolve these issues, allow maintaining the stored auth-method, `Sign out` should go to the `Your keys` (multiaccount overview) screen.

See future steps, current key encryption and storage needs to be investigated. If keys are encrypted with the password, rather than a key generated by status-react (as proposed [here](https://docs.google.com/document/d/1IrjvvAuAQbYsr3xqZvXfgAeeyDZRl_x9KRxWnLcwVuI/edit#bookmark=id.h09ezqb1viei)), credentials (password?) might be required. Possibly auth-method is erased to prompt password.

#### Desired flow
Sign out > Select keys to unlock > Unlock screen
![Frame 26](https://user-images.githubusercontent.com/35233054/73058376-c2f25f00-3e93-11ea-8265-ff0497ffb046.png)

## Acceptance Criteria
`Sign out` opens `Your keys`

## Future Steps
- Depending on how keys are encrypted and stored, create issue to investigate key encryption according to [proposal](https://docs.google.com/document/d/1IrjvvAuAQbYsr3xqZvXfgAeeyDZRl_x9KRxWnLcwVuI/edit#bookmark=id.h09ezqb1viei)
- Depending on whether keys can be encrypted and stored according to proposal, investigate maintaining stored auth-method

**Comment by hesterbruikman** [comment]
@errorists @andremedeiros @Ferossgp I find this a pretty complex issue as it feels like a bandage on top of a bandage. Would appreciate your eyes to make sure this issue and future steps are clear.

**Comment by hesterbruikman** [comment]
@Ferossgp it might make sense to bundle this issue with #9782 which is purely visual UI updates. Take a look please to see if that makes sense.

**Comment by Ferossgp** [comment]
@hesterbruikman I would keep them implemented as separate PR, it will be easier to test and review that way.

**Comment by Ferossgp** [comment]
@hesterbruikman could you please confirm that it should open multi-account only after logout, and it does not affect the screen which appears right before app open? Right now they are the same, so I'm wondering if it should change bothflows.

**Comment by hesterbruikman** [comment]
Confirming that this is only for the screen after logout.

- Log out > Your keys
- Open app > (Behavior as is) Last opened multiaccount unlock screen (means the multiaccount opens automatically if 'save password' is set). 

Thanks for checking!
